src/index.ts

```diff
@@ -177,8 +177,8 @@ function sortAst(
 export const parsers = {
   json: {
     ...babelParsers.json,
-    parse(text, _parsers, options: any) {
-      const ast: Expression = babelParsers.json.parse(text, _parsers, options);
+    parse(text, options: any) {
+      const ast: Expression = babelParsers.json.parse(text, options);
```

Comment:
Better await the parse result, we may change it in the future.

Reply:
Thanks! Will do